cleanlab/filter.py

```diff
@@ -290,38 +281,43 @@ def get_noise_indices(
 
     num_to_remove_per_class : list of int of length K (# of classes)
       e.g. if K = 3, num_to_remove_per_class = [5, 0, 1] would return
-      the indices of the 5 most likely mislabeled examples in class s = 0,
-      and the most likely mislabeled example in class s = 1.
+      the indices of the 5 most likely mislabeled examples in class labels = 0,
+      and the most likely mislabeled example in class labels = 1.
 
       Note
       ----
-      Only set this parameter if ``prune_method == 'prune_by_class'``
-      You may use with ``prune_method == 'prune_by_noise_rate'``, but
+      Only set this parameter if ``filter_by == 'prune_by_class'``
+      You may use with ``filter_by == 'prune_by_noise_rate'``, but
       if ``num_to_remove_per_class == k``, then either k-1, k, or k+1
       examples may be removed for any class. This is because noise rates
       are floats, and rounding may cause a one-off. If you need exactly
       'k' examples removed from every class, you should use ``'prune_by_class'``
 
-    prune_method : str (default: 'prune_by_noise_rate')
-      Possible Values: 'prune_by_class', 'prune_by_noise_rate', or 'both'.
-      Method used for pruning.
+    filter_by : str (default: 'prune_by_noise_rate')  TODO: change default to cl_off_diag?
+      Possible Values: {'prune_by_class', 'prune_by_noise_rate', 'both',
+                        'confident_learning', 'argmax_not_equal_given_label'}
+      Method used for filtering/pruning out the label issues.
```

Comment:
descriptions of each option should be standardized as to what is being "removed" or "found". Currently some descriptions say "find" (eg. "confident_learning"), others say "remove" (eg. options 1 and 2).